client/components/NavBar.tsx

```diff
@@ -51,50 +66,136 @@ const LogoContainer = styled.div`
 
 const LogoW = styled.img`
   display: none;
-  @media (max-width: 425px) {
+  @media ${smallerThan(width.tablet)} {
     display: block;
   }
+  padding: 14px 24px;
 `
+
 LogoW.defaultProps = {
   src: '/W.svg',
   alt: 'Waterpark logo',
 }
 
 const LogoWaterpark = styled.img`
   display: block;
-  @media (max-width: 425px) {
+  @media ${smallerThan(width.tablet)} {
     display: none;
   }
+  padding: 14px 24px;
 `
+
 LogoWaterpark.defaultProps = {
   src: '/Waterpark.svg',
   alt: 'Waterpark logo',
 }
 
-export const NavBar = () => {
+const NavbarItemsStyling = styled.div`
+  display: flex;
+  justify-content: space-between;
+
+  @media ${smallerThan(width.mobile)} {
+    flex-direction: column;
+    align-items: flex-end;
+  }
+`
+
+const NavBarContainer = styled.div`
+  position: fixed;
+  top: 0;
+  z-index: 1;
```

Comment:
See https://github.com/loolabs/waterpark/pull/177#issuecomment-881857536

```suggestion
  z-index: 8;
```

Reply:
Nice catch!